service/journal/mapping.go

```diff
@@ -22,3 +23,30 @@ func MapRepoSliceToVMSlice(js []repo.Journal) []models.JournalVM {
 
 	return jVMs
 }
+
+func RepoToPageOfVMs(js []repo.Journal, params models.JournalQueryParams) models.PageOfJournalVMs {
+	total := len(js)
+	var page int
+	var size int
+
+	if params.Page <= 0 {
+		page = 1
+	} else {
+		page = params.Page
+	}
+
+	if params.Size <= 0 {
```

Comment:
issue: you actually allow for size to be zero during validation. So either you need to prevent that or update this to be a pure less-than

Reply:
actually these if statements shouldn't be necessary since you are validating them ahead of time anyway